tools/risk-form-filler/src/object-models/sections.ts

```
@@ -0,0 +1,588 @@
+      openFlames: new Field(
+        'emergency_flames',
+        'string',
+        'Are there any open flames or dust-producing activities as part of this event?',
+        true,
+        ['No', 'Yes'],
+      ),
+    }
+  }
+}
+
+// Safety Risk Assessment
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)